apps/www/components/framework-docs.tsx

```diff
@@ -0,0 +1,25 @@
+"use client"
+
+import * as React from "react"
+import { Mdx } from "./mdx-components"
+import { allDocs } from "contentlayer/generated"
+
+interface FrameworkDocsProps extends React.HTMLAttributes<HTMLDivElement> {
+  data: string
+}
+
+export function FrameworkDocs({
+  ...props
+}: FrameworkDocsProps) {
+  const frameworkDoc = allDocs.find((doc) => doc.slug === `/docs/frameworks/${props.data}`)
```

Comment:
@dan5py I don't really love this because this kinda makes these docs a part of the same set of pages that can load as a part of of the regular docs slug page.

Reply:
That's not really that bad. I mean you can see the docs for a frameworks directly and also share the link for it.